config/initializers/limits.rb

```diff
@@ -2,3 +2,11 @@
 def secrets_version_limit
   ( ENV['SECRETS_VERSION_LIMIT'] || 20 ).to_i
 end
+
+# Max size of policies that can be loaded (or data sent to POST endpoints in
+# general)
+Rack::Utils.key_space_limit = 1048576 * 10 # 10 MiB
+
+# This sets the default size of a Rack::Utils::KeySpaceConstrainedParam, which
```

Comment:
This comment should get combined with the one above it. Having them separate, and having a comment after the setting it's documenting, makes it harder to understand.